toml.abnf

```diff
@@ -31,6 +31,7 @@ wschar =/ %x09  ; Horizontal tab
 
 newline =  %x0A     ; LF
 newline =/ %x0D.0A  ; CRLF
+newline =/ %x0D     ; CR
```

Comment:
Not sure if there's a good way to say "followed by EOF or any codepoint except LF", or if that's even necessary.

Reply:
newline =/ %x0D [%0A]  ; CR or CRLF

That should work, I think?

Reply:
Oh, thanks, how did I miss that.

The ABNF spec doesn't seem to mention anything about the greediness of that expression, so wondering if a CRLF could in theory match two `newline`s instead of one? Note that I think the same problem may apply to current solution using alternatives, because AFAIK alternatives are unordered.

There is a place where this could potentially be an issue, and that is the single optional newline that prefixes MLB and MLL strings. That said, I'm not sure if it's necessary for the ABNF to explicitly express that newline in the way it currently does.